main.js

```diff
@@ -10,3 +19,26 @@ function dayOfWeek(event) {
   var sentence = event.target.textContent;
   schedule.textContent = 'Scheduled Events for ' + sentence;
 }
+
+function entryModal() {
+  modal.classList.remove('hidden');
+}
+window.onclick = function (event) {
+  if (event.target === modal) {
+    modal.classList.add('hidden');
+  }
+};
+
+function addTask(event) {
+  for (var i = 1; i < tableRow.length; i++) {
+    if (!tableRow[i].firstElementChild.textContent && !tableRow[i].lastElementChild.textContent) {
+      tableRow[i].firstElementChild.textContent = entryTime.value;
+      tableRow[i].lastElementChild.textContent = entryDescription.value;
+
```

Comment:
By default you will not have any elements in your table, you can create new elements instead of modifying the existing ones